packages/lib/test/defaultStyles.spec.js

```diff
@@ -8,7 +8,7 @@ describe('defaultStyles', () => {
     expect(_.isEqual(actual, expectedStyles())).eq(true);
     const actualLength = _.keys(actual).length;
     expect(actualLength).eq(_.keys(expectedStyles()).length);
-    expect(actualLength).eq(90);
+    expect(actualLength).eq(89);
```

Comment:
@nirnaor , do we want thins hardcoded?

Reply:
I prefer to keep it hard coded fow now. I want us to track additions/removals of styles manually (for now)